src/biotite/database/uniprot/query.py

```diff
@@ -165,6 +165,7 @@ def search(query, number=10):
         The search query.
     number : int
         The maximum number of IDs that are obtained.
+        (Default: 10)
```

Comment:
I would suggest to set this parameter to 500 as this seems to be the new maximum value of a single request (taken from https://www.uniprot.org/help/api_queries):

> Always use size=500 as this will provide fast performance.